mobile/lib/modules/archive/views/archive_page.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/modules/home/ui/asset_grid/immich_asset_grid.dart';
+import 'package:immich_mobile/shared/models/asset.dart';
+import 'package:immich_mobile/shared/providers/db.provider.dart';
+import 'package:isar/isar.dart';
+
+class ArchivePage extends HookConsumerWidget {
+  const ArchivePage({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final archiveAssets = ref
+        .watch(dbProvider)
```

Comment:
@fyfrey I have trouble with this one on re-render the assets on the page after `unarchive` an asset. The unarchive asset does not update. Do you have any suggestions?

Reply:
I've reworked this to listen on DB changes and re-render the assets. Re-rendering is quite slow (possibly because renderList of the asset grid is not used), but we can improve that in another PR